ts/editor-toolbar/dropdown.scss

```diff
@@ -0,0 +1,4 @@
+@import "ts/node_modules/bootstrap/scss/functions";
+@import "ts/node_modules/bootstrap/scss/variables";
+@import "ts/node_modules/bootstrap/scss/mixins";
+@import "ts/node_modules/bootstrap/scss/dropdown";
```

Comment:
Most of the ts/svelte code I pushed in https://github.com/ankitects/anki/commit/7f738c11a2fbf54d7a965d7eaa22f98cc8862f1d is just me fumbling my way through and is probably not best practice :-) But one thing you may want to adopt is the way I've added a ts/bootstrap folder that exposes a sass library - that way you can add it as a dependency and import from it cleanly:

https://github.com/ankitects/anki/commit/7f738c11a2fbf54d7a965d7eaa22f98cc8862f1d#diff-3dfaaa14709dfa6fec287bccf73739804b971790e23d6b24dea8661c7bacd6d2R14
https://github.com/ankitects/anki/commit/7f738c11a2fbf54d7a965d7eaa22f98cc8862f1d#diff-eeb95d9e3e56521ca955e5a954d0d63e0063981a63bdf62404b130151e6860ebR4